src/app/garfile/supportingdocuments/get.controller.js

```diff
@@ -22,6 +22,7 @@ module.exports = (req, res) => {
       switch (req.query.query) {
         case 'e': // Error returned
           error = [req.session.errMsg];
+          delete req.session.errMsg;
```

Comment:
Question: What issue was caused by this change and how does it fix this?
- This question is out of curiosity.
- I presume that by deleting the property, it does not display the supporting document's `req.session.errMsg` message else where e.g. another form like custom section or responsible people.
- However, I wanted to check so I could understand it better.

Reply:
In one of the scenarios while trying to upload a file to the an already submitted GAR, the error message displayed stays if click on "Aircraft" tab. I think that is because the error message stays in the session. As it is cleared in other parts of the code, I thought of clearing it not to display the error message if moved to other pages.